packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentActions.tsx

```diff
@@ -552,6 +563,23 @@ const UpdateAction: DocumentActionComponent = ({ activeTab, id, model, collectio
             },
             document
           );
+        } else if (isCloning) {
+          const res = await clone(
+            {
+              model,
+              id: cloneMatch.params.origin!,
+            },
+            document
+          );
+
+          if ('data' in res) {
+            /**
+             * TODO: refactor the router so we can just do `../${res.data.id}` instead of this.
+             */
+            navigate({
+              pathname: `../${collectionType}/${model}/${res.data.id}`,
+            });
+          }
```

Comment:
I'm not sure but should the isCloning condition come first? Is there any case
when we're on the clone route that we would want to call the normal update?

Reply:
I'm not sure it should make a huge difference, but cognitively it probably makes more sense the way round you're proposing. Lets do that.